.gitignore

```diff
@@ -1,3 +1,8 @@
+.DS_Store
+*.iml
+.classpath
+.project
+.idea/
```

Comment:
What do you mean with "without success"? They're perfectly fine ignored now and it makes sense, not everybody uses Eclipse, even though I did for years I prefer IntelliJ these days. And it is weird, having Maven directories excluded (makes perfect sense) but not those useless IDE files (there's another PR where somebody thinks Eclipse files belong into the repo and NO they don't, they should be excluded by default :-))